src/pages/clients-list/index.js

```diff
@@ -588,10 +588,12 @@ console.log(userData)
   const WorkAddressValidation = useFormik({
     enableReinitialize: false,
     validateOnChange: false,
+    validateOnBlur:true,
     validate : (values) => {
       const errors = {};
+      const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
 
-      if (values.name || values.name || values.phone || values.countryId ||  values.street1)  {
+      if (values.name || values.street1 || values.phone || values.countryId ||  values.street1)  {
```

Comment:
values.street1 is added twice

Reply:
this fixed in other Branch also i fixed now